### PR TITLE
Add support for mesos exporter

### PR DESCRIPTION
**Pull request by tahaalibra**
This PR adds support for mesos exporter. Please review

### CODE REVIEW

File: manifests/mesos_exporter.pp [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  $package_name         = $::prometheus::params::mesos_exporter_package_name,
+  $purge_config_dir     = true,
+  $restart_on_change    = true,
+  $service_enable       = true,
+  $service_ensure       = 'running',
+  $user                 = $::prometheus::params::mesos_exporter_user,
+  $version              = $::prometheus::params::mesos_exporter_version,
+) inherits prometheus::params {
+  $real_download_url    = pick($download_url,"${download_url_base}/download/v${version}/${package_name}-${version}.${os}-${arch}.${download_extension}")
+  validate_bool($purge_config_dir)
```
Comment (bastelfreak) [inline manifests/mesos_exporter.pp]: Could you replace those validate function calls with native puppet datatypes?

**Comment by tahaalibra** [comment]
@bastelfreak now using inbuilt puppet datatype

**Comment by bastelfreak** [comment]
Sorry, I was a bit unclear. Please add datatypes to all params. There are multiple classes in this module that you can use as an example. Please also check the used email address in your commit. It isn't associated with your github account/

**Comment by tahaalibra** [comment]
@bastelfreak hi, yea i was thinking that maybe datatype should be added to other params. I added the params also a squashed the commit, i hope that will be fine

**Comment by bastelfreak** [comment]
Can you take a look at the failing acceptance test?

**Comment by tahaalibra** [comment]
yea sure

**Comment by tuxmea** [comment]
I restarted the failing job. now tests are green.

**Comment by tahaalibra** [comment]
thanks @tuxmea

**Review by juniorsysadmin — APPROVED** [review]

**Pull request merged**